src/common/libutil/test/stdlog.c

```diff
@@ -127,28 +127,57 @@ int main(int argc, char** argv)
         "stdlog_decode decoded procid") ;
     ok (streq (hdr.msgid, cln.msgid),
         "stdlog_decode decoded msgid") ;
-    ok (sdlen == strlen (STDLOG_NILVALUE) && strncmp (sd, STDLOG_NILVALUE, sdlen) == 0,
+    ok (sdlen == strlen (STDLOG_NILVALUE)
+        && strncmp (sd, STDLOG_NILVALUE, sdlen) == 0,
         "stdlog_decode decoded structured data");
-    ok (msglen == strlen (STDLOG_NILVALUE) && strncmp (msg, STDLOG_NILVALUE, msglen) == 0,
+    ok (msglen == strlen (STDLOG_NILVALUE)
+        && strncmp (msg, STDLOG_NILVALUE, msglen) == 0,
         "stdlog_decode decoded message");
```

Comment:
code re-formatting accidental?  Or supposed to in another commit?

Reply:
Yes, I did forget to split this code formatting update into a separate commit.